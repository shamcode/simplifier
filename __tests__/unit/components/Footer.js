import Footer from '../../../src/components/Footer.sht';
import renderer from 'sham-ui-test-helpers';

it( 'renders correctly', () => {
    const meta = renderer( Footer, {} );
    expect( meta.toJSON() ).toMatchSnapshot();
} );
