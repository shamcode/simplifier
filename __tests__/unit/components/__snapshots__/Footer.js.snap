// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Object {
  "Constructor": "Footer",
  "Options": Object {
    "ID": "component",
    "containerSelector": "body",
    "types": Array [],
  },
  "html": "<footer></footer>",
}
`;
